apps/desktop/electron/database/DatabaseActions.ts

```diff
@@ -14,41 +13,6 @@ export interface DatabaseResponse<DatabaseItemType> {
     readonly data: DatabaseItemType;
 }
 
-/**
- * Decrement all of the undo actions in the most recent group down by one.
- *
- * This should be used when a database action should not create its own group, but the group number
- * was incremented to allow for rolling back changes due to error.
- *
- * @param db database connection
- */
-function decrementLastUndoGroup(db: Database.Database) {
```

Comment:
This function was unused, and should've been deleted in my last pr. Unrelated to drizzle